sdk/loadtestservice/Azure.Analytics.LoadTestService/src/Models/AppComponentModel.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Why there are models?

Reply:
For easy serialization and deserialization, I created them.

Reply:
If it is test purpose, remember to remove it right before the review is ready. If it is customization purpose, please follow https://github.com/Azure/azure-sdk-for-net/blob/e56a0f677e07dd068626af324b8a0f63fa957fcc/doc/DataPlaneCodeGeneration/DeveloperDrivenEvolution.md#L1 to add convenience API.